control/src/test/java/org/kaazing/k3po/control/internal/ControlTest.java

```diff
@@ -127,6 +127,7 @@ public void shouldWritePrepareCommand() throws Exception {
         final byte[] expectedPrepare =
                 ("PREPARE\n" +
                  "version:2.0\n" +
+                 "content-length:0\n" +
```

Comment:
I don't think this is backwards incompatible, at least it should not be.  And this test should be testing that: https://github.com/k3po/k3po/blob/develop/specification/k3po.control/src/main/scripts/org/kaazing/specification/control/accept.finished.empty.rpt